net-ldap.gemspec

```diff
@@ -31,4 +31,5 @@ the most recent LDAP RFCs (4510-4519, plutions of 4520-4532).}
 
   s.add_development_dependency("flexmock", "~> 1.3")
   s.add_development_dependency("rake", "~> 10.0")
+  s.add_development_dependency("pry")
```

Comment:
:-1: as much as I like `pry`, I don't think this is a necessary development dependency.